### PR TITLE
Add Warning check in updateSchema command output

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | I added a condition to see if warnings are present in the result of the command and if we are in a development environment
| Type?             | improvement
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | see https://github.com/PrestaShop/PrestaShop/issues/32547
| Fixed ticket?     | https://github.com/PrestaShop/PrestaShop/issues/32547
| Related PRs       | -
| Sponsor company   | -

### CODE REVIEW

File: src/PrestaShopBundle/Install/Install.php [hunk 1]
```diff
@@ -346,9 +346,16 @@ public function updateSchema()
         $schemaUpgrade = new UpgradeDatabase();
         $schemaUpgrade->addDoctrineSchemaUpdate();
         $output = $schemaUpgrade->execute();
+        $schemaUpdateOutput = $output['prestashop:schema:update-without-foreign']['output'];
+        if (_PS_MODE_DEV_ && substr($schemaUpdateOutput, 2, 9) === '[WARNING]') {
```
Comment (kpodemski) [inline src/PrestaShopBundle/Install/Install.php]:
```suggestion
        if ($this->isDebug && substr($schemaUpdateOutput, 2, 9) === '[WARNING]') {
```

Comment (M0rgan01) [reply src/PrestaShopBundle/Install/Install.php]: I had not seen the variable :) thank you

**Review by boherm — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by AureRita** [comment]
Hello @M0rgan01, 

as the issue isn't reproduce, we can test it. However, because this one need to be reproduced by a dev, @PrestaShop/prestashop-core-developers  can someone try to reproduce the issue and see if this PR correct them ? 

Thank you !

**Comment by boherm** [comment]
It's good for me after testing, thanks @M0rgan01!

**Pull request merged**